#1#travis.log

```diff
@@ -0,0 +1,87 @@
+Command line:
```

Comment:
What does this file do @xcixor ?

Reply:
that travis.log is redundant I had been trying a different method of reporting coverage but abandoned it, let me remove that. As for pylint I did set it up and also included it in my requirements.txt